src/blocks/modal.css

```diff
@@ -1,216 +1,198 @@
-  top: -40px;
-  right: -40px;
+  height: 40px;
+  width: 40px;
+  bottom: calc(100% + 5px);
+  left: calc(100% + 5px);
+  border: none;
+  background: transparent no-repeat url(../images/Close\ Icon.svg) center /
+    contain;
+  transition: ease 456ms opacity;
   cursor: pointer;
 }
-
 .modal__close:hover {
   opacity: 0.6;
-  /* transform: rotate(45deg); */
-}
-
-.modal__image {
-  /* position: fixed; */
-  /* top: 50%;
-  left: 50%; */
-  /* transform: translate(-50%, -50%); */
-  max-width: 75vw;
-  max-height: 75vh;
 }
-
-.modal__text {
-  color: #ffffff;
-  /* position: absolute; */
-  width: 291px;
-  height: 12px;
-  top: 640px;
-  left: 232px;
-  font-size: 12px;
-}
-
-.modal__content_card {
-  width: auto;
-  height: auto;
-  border-radius: 0;
-  background: transparent;
-  max-width: 75vw;
-  max-height: 75vh;
+.modal__title {
+  max-width: 330px;
+  font-family: "Inter", Arial, sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 24px;
+  line-height: 29px;
+  color: #000000;
+  margin: 0 0 24px 0;
 }
-
-/* .form__label {
+.modal__form {
   display: flex;
   flex-direction: column;
+  row-gap: 30px;
+}
+.modal__label {
+  max-width: 385px;
+  height: 44px;
+  position: relative;
+}
+.modal__input {
+  width: 100%;
+  font-family: "Inter", Arial, sans-serif;
+  height: auto;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 17px;
+  color: #000000;
   border: none;
-  font-family: "Inter", Arial, Helvetica, sans-serif;
-} */
-
-/* .form__fieldset {
-  font-family: "inter", Arial, Helvetica, sans-serif;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.2);
+}
+.modal__button {
+  width: 358px;
+  height: 50.15px;
+  background-color: rgba(0, 0, 0, 1);
+  color: #ffffff;
+  border-radius: 2px;
   padding: 0;
-  border: none;
-  margin: 0;
-  margin-left: 36px;
-} */
-
-/* .form__button {
-  background-color: #000;
-  color: #fff;
+  margin: 18px 0 0 0;
+  transition: background-color ease 600ms;
   border: none;
   cursor: pointer;
-  transition: 0.5;
-} */
-
-@media screen and (min-width: 320px) and (max-width: 544px) {
-  /* .modal__name__item {
-    width: 238px;
-    height: 27px;
-    font-size: 14px;
-  } */
-
-  .modal__content {
-    width: 282px;
-    height: 322px;
+  font-family: "Inter", Arial, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 22px;
+  text-align: center;
+}
+.modal__button:hover {
+  background-color: rgba(0, 0, 0, 0.8);
+}
+.modal__error_visible {
+  visibility: visible;
+}
+.modal__error {
+  position: absolute;
+  bottom: -20px;
+  height: auto;
+  left: 0;
+  width: 100%;
+  font-family: "Inter";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 15px;
+  color: #ff0000;
+  margin: 5px 0 0 0;
+  padding: 0;
+}
+.modal__button_type_error {
+  border-bottom: 1px solid #ff0000;
+}
+.modal__button_disabled {
+  background-color: #fff;
+  color: grey;
+  border: 1px solid grey;
+}
```

Comment:
The cursor of a disabled button should not be `pointer`, use `default`

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

Reply:
Don't forget to fix this